src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -77,10 +77,13 @@ def update_github_repo(self):
         run_check(['git', 'config', '--global', 'http.postBuffer', '12M'])
         repo_url = 'https://%s:%s@github.com/marco-c/gecko-dev' % (self.gecko_dev_user, self.gecko_dev_pwd)
         repo_path = os.path.join(self.cache_root, 'gecko-dev')
-        if not os.path.isdir(repo_path):
-            run_check(['git', 'clone', repo_url], cwd=self.cache_root)
-        run_check(['git', 'pull', 'https://github.com/mozilla/gecko-dev', 'master'], cwd=repo_path)
-        run_check(['git', 'push', repo_url, 'master'], cwd=repo_path)
+        try:
+            if not os.path.isdir(repo_path):
+                run_check(['git', 'clone', repo_url], cwd=self.cache_root)
+            run_check(['git', 'pull', 'https://github.com/mozilla/gecko-dev', 'master'], cwd=repo_path)
+            run_check(['git', 'push', repo_url, 'master'], cwd=repo_path)
+        except:
+            pass
```

Comment:
maybe it would be better to have some kind of a retry logic or at least to log the exception. feels weird to just swallow the exception.